.gitignore

```diff
@@ -103,6 +103,7 @@ images/scorecard-test/scorecard-test
 pkg/ansible/runner/testdata/valid.yaml
 /bin
 test/test-framework/go.sum
+internal/scorecard/alpha/examples/custom-scorecard-tests/go.sum
 
 # Website
 website/public/
```

Comment:
should we not add a fragment for that?

Reply:
I don't think so, shouldn't have any impact on the user. Fragment probably required for some other stuff htough

Reply:
is not it changing the json output? see [here](https://github.com/operator-framework/operator-sdk/pull/3208/files#diff-b86df888a60472475e24887d92f4c4aaL162-R176)

Reply:
It looks like that link just takes me back to this conversation